src/view/GameWindow.java

```diff
@@ -2,15 +2,43 @@
 
 import model.GameBoard;
 
+import java.awt.Dimension;
+
 import javax.swing.JFrame;
+import javax.swing.JScrollPane;
 
 public class GameWindow extends JFrame {
-    private final GameBoard gameBoard = new GameBoard();
+    private final GameBoard gameBoard = new GameBoard(90, 90);
+    DrawArea drawArea;
+    int drawAreaX = 2880;
```

Comment:
drawAreaX/Y sizes could be defined in the constants class

Reply:
In my version since it was made before the constant class, I can't define it there just yet but it is definitely a good idea and I will implement it in the future. Thank you for the idea.